components/connector-service/internal/monitoring/collector/summarycollector.go

```diff
@@ -0,0 +1,29 @@
+package collector
+
+import (
+	"github.com/kyma-project/kyma/components/connector-service/internal/apperrors"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type Collector interface {
+	AddObservation(message float64, labelValues ...string)
```

Comment:
I would rename message parameter (it's rather something like observation or value)

Reply:
Done